sklearn/preprocessing/tests/test_data.py

```diff
@@ -691,7 +691,7 @@ def test_standard_check_array_of_inverse_transform():
 )
 @pytest.mark.parametrize(
     "estimator",
-    [MaxAbsScaler(), MinMaxScaler()],
+    [MaxAbsScaler(), MinMaxScaler(), Binarizer()],
     ids=_get_check_estimator_ids,
 )
 def test_scaler_array_api_compliance(estimator, check, array_namespace, device, dtype):
```

Comment:
```suggestion
def test_preprocessing_array_api_compliance(estimator, check, array_namespace, device, dtype):
```
what do you think of renaming this? It seems like it is more general than just scalers.